pkg/driver/driver.go

```diff
@@ -15,15 +14,22 @@ import (
 	"time"
 )
 
-const MySQL = "icingadb-mysql"
-const PostgreSQL = "icingadb-pgsql"
+const MySQL = "mysql"
+const PostgreSQL = "pgsql"
```

Comment:
Previously, those were custom driver names so that we could use `sqlx.Open()`. Now, only one matches the driver name we use in the background, the other doesn't. Also note that if we simply tell sqlx that our driver is `"postgres"`, we don't even need the `sqlx.BindDriver()` call.

I'd change this to `PostgreSQL = "postgres"` (or maybe we don't need them at all, they might already exist as constants in the driver packages?).

Also, these two cases elsewhere now simply look like something was forgotten:
```go
case driver.MySQL:
case driver.PostgreSQL, "postgres":
```

Reply:
> or maybe we don't need them at all, they might already exist as constants in the driver packages?

I couldn't find such constants neither in [`pq`](https://pkg.go.dev/github.com/lib/pq#pkg-constants) nor in [`mysql`](https://pkg.go.dev/github.com/go-sql-driver/mysql#pkg-constants).

Reply:
Sure they do:

`./github.com/go-sql-driver/mysql@v1.8.0/driver.go:96: sql.Register(driverName, &MySQLDriver{})`

`./github.com/lib/pq@v1.1.1/conn.go:52: sql.Register("postgres", &Driver{})`

But [pgx](https://github.com/jackc/pgx) registers as `pgx`.

Reply:
I would also try to get rid of the constants.

Reply:
Sorry, I thought you were simply talking about the driver names! Still, I think relying on `mysql` and `postgres` as strings is good enough.

Reply:
As discussed offline, we opt to keep the constants in order to maintain the driver names in a centralized place.